src/zenml/integrations/evidently/data_validators/evidently_data_validator.py

```diff
@@ -0,0 +1,156 @@
+    "classificationmodelperformance": ClassificationPerformanceTab,
+    "regressionmodelperformance": RegressionPerformanceTab,
+    "probabilisticmodelperformance": ProbClassificationPerformanceTab,
+}
+
+
+def get_profile_sections_and_tabs(
+    profile_list: Optional[Sequence[str]],
+    verbose_level: int = 1,
+) -> Tuple[List[ProfileSection], List[Tab]]:
+    """Get the profile sections and dashboard tabs for a profile list.
+
+    Args:
+        profile_list: List of identifiers for Evidently profiles.
+        verbose_level: Verbosity level for the rendered dashboard. Use
+            0 for a brief dashboard, 1 for a detailed dashboard.
+
+    Returns:
+        A tuple of two lists of profile sections and tabs.
+
+    Raises:
+        ValueError: if the profile_section is not supported.
+    """
+    profile_list = profile_list or list(profile_mapper.keys())
+    try:
+        return (
+            [profile_mapper[profile]() for profile in profile_list],
+            [
+                dashboard_mapper[profile](verbose_level=verbose_level)
+                for profile in profile_list
+            ],
+        )
+    except KeyError as e:
+        nl = "\n"
+        raise ValueError(
+            f"Invalid profile sections: {profile_list} \n\n"
+            f"Valid and supported options are: {nl}- "
+            f'{f"{nl}- ".join(list(profile_mapper.keys()))}'
+        ) from e
+
+
+class EvidentlyDataValidator(BaseDataValidator):
+    """Evidently data validator stack component."""
+
+    # Class Configuration
+    FLAVOR: ClassVar[str] = EVIDENTLY_DATA_VALIDATOR_FLAVOR
+    NAME: ClassVar[str] = "Evidently"
+
+    def data_profiling(
+        self,
+        dataset: pd.DataFrame,
+        comparison_dataset: Optional[pd.DataFrame] = None,
+        profile_list: Optional[Sequence[str]] = None,
+        column_mapping: Optional[ColumnMapping] = None,
+        verbose_level: int = 1,
+        **kwargs: Any,
+    ) -> Tuple[Profile, Dashboard]:
+        """Analyze a dataset and generate a data profile with Evidently.
+
+        Args:
+            dataset: Target dataset to be profiled.
+            comparison_dataset: Optional dataset to be used for data profiles
+                that require a baseline for comparison (e.g data drift profiles).
+            profile_list: Optional list identifying the categories of Evidently
+                data profiles to be generated.
+            column_mapping: Properties of the DataFrame columns used
+            verbose_level: Level of verbosity for the Evidently dashboards. Use
+                0 for a brief dashboard, 1 for a detailed dashboard.
+            **kwargs: Extra keyword arguments (unused).
+
+        Returns:
+            The Evidently Profile and Dashboard objects corresponding to the set
+            of generated profiles.
+        """
+        sections, tabs = get_profile_sections_and_tabs(
+            profile_list, verbose_level
+        )
+        dashboard = Dashboard(tabs=tabs)
```

Comment:
Wondering whether you could include the new colour schema customisation that I worked on? https://docs.evidentlyai.com/user-guide/customization/options-for-color-schema shows how it works. You can see at the bottom there are some pre-defined color schemas that you can use etc.

Forget it if it's too much work, but would be nice to offer it as an option.

As a broader point, I think it might be worth considering adding an `options` list on the config which we can then just add into the `Dashboard` on initialisation. It's a fairly common behaviour to define custom tests etc, as far as I'm aware (see the docs for more [here](https://docs.evidentlyai.com/user-guide/customization)), and without us allowing users to pass in those options they'd be out of luck with our integration AFAICS?

Reply:
I see what you mean, I did the same for Deepchecks. I'll see if I can do the same here, should be pretty straightforward.

Reply:
Not as straightforward as I imagined, but it's done. Added options for both dashboard and profile, just in case.